specification/window.browser.bs

```diff
@@ -0,0 +1,57 @@
+<pre class="metadata">
+Title: window.browser
+Shortname: wecg-browser
+Level: 1
+Group: wecg
+Status: UD
+URL: https://patrickkettner.github.io/window.browser
+Editor: Patrick Kettner, Google, patrickkettner@google.com
+Abstract: This specification reserves the <code>window.browser</code> namespace for use by WebExtensions.
+Repository: patrickkettner/window.browser
+Complain About: accidental-2119 on, missing-example-ids on
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:attribute; text:window
+</pre>
+
+<pre class="anchors">
+spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
+    type: dfn; text: valid non-empty URL; url: valid-non-empty-url-potentially-surrounded-by-spaces
+</pre>
+
+<section class="non-normative">
+  ## Introduction {#introduction}
+  <em>This section is non-normative.</em>
+
+  Modern WebExtensions are an evolution of a system originally introduced in
```

Comment:
Question: Is this sufficient, or do we need a very brief introduction to what "WebExtensions" is, to people unfamiliar with WebExtensions?

Originally, it was just "Chrome Extensions". When Mozilla adopted the same model, the name "WebExtensions" was coined, described at https://wiki.mozilla.org/Add-ons/Terminology. Although that page asserts that "WebExtensions" ought to refer to the technology, and not the concrete extensions built with it, in practice the term is being used to refer to both.

Here is a small suggestion to add some context, and to also call out why there is a need for a "browser" namespace at all.

```suggestion
  Browser extensions are programs to extend the capabilities of web browsers.
  Although not standardized at first, web browsers have independently implemented a common interoperable set of extension APIs, called WebExtensions.
  
  Modern WebExtensions are an evolution of a system originally introduced in
```